src/lib/resources/base.resource.ts

```diff
@@ -22,6 +22,17 @@ export class BaseResource {
     return response.Results[0].data[0][voName];
   }
 
+  getVosFromResponse<T = PermanentApiResponseDataBase>(
+    response: PermanentApiResponse<T>,
+    voName: keyof T
+  ) {
+    let voArray: T[keyof T][] = [];
+    response.Results[0].data.forEach(element => {
+      voArray.push(element[voName]);
+    });
+    return voArray;
+  };
```

Comment:
I don't mean to code golf, but the "declare new array; iterate old array & push onto new array; return new array" pattern is a bit unidiomatic. Instead, you can use [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map):

```suggestion
  getVosFromResponse<T = PermanentApiResponseDataBase>(
    response: PermanentApiResponse<T>,
    voName: keyof T
  ): T[keyof T][] {
    return response.Results[0].data.map(element => element[voName]);
  };
```

Reply:
Oh great, thanks.  I think the "make array and add items to it" is a php-ism.